Assets/Scripts/Entities/Items/Item.cs

```diff
@@ -1,9 +1,60 @@
-﻿namespace Assets.Scripts.Entities.Items
+﻿using Assets.Scripts.Entities.Components;
+using System;
+using UnityEngine;
+
+namespace Assets.Scripts.Entities.Items
 {
-    abstract class Item
+    class Item
     {
-        public int id { get; private set; }
-        public string name { get; private set; }
-        public int coinValue { get; private set; }
+        /// <summary>
+        /// Path on disk where item data is
+        /// </summary>
+        public static readonly string ITEM_PATH = $"{Application.dataPath}/Resources/data/item_data/items.json";
+        /// <summary>
+        /// Path on disk where item icons are
+        /// </summary>
+        public static readonly string ICON_PATH = "textures/icon_textures/item_icons";
+
+        public int id { get; set; }
+        public Renderable iconSprite;
+        public string name { get; set; }
+        public int buyValue { get; set; }
+        public int selValue { get; set; }
+        public string description { get; set; }
+
+        /// <summary>
+        /// Used to polymorphically build special attrivutes of item sub-types
+        /// </summary>
+        /// <param name="itemBuilder">The itembuilder which holds the reference to the item's members</param>
+        internal virtual void build(ItemBuilder itemBuilder){}
     }
+
+    class ItemFactory
+    {
+        /// <summary>
+        /// Factory class for creating new item types.
+        /// </summary>
+        /// <remarks>
+        /// Item type that is consumed on usage. Note: Consumable items <b>are Useable</b> but Useable Items aren't always Consumeable.
+        /// </remarks>
+        /// <param name="isConsumeable"></param>
+        /// <param name="isUseable">hey</param>
+        /// <returns></returns>
+        public static Item create(bool isConsumeable, bool isUseable)
+        {
+            if (isConsumeable)
+            {
+                return new Consumable();
+            }
+            else if(isUseable)
+            {
+                return new Equipable();
```

Comment:
should this be isEquipable?

Reply:
Hmm, I suppose this comes down to how you want to express how something is `equipable`.

So far `equipable = ~Conseumable ^  Useable `, And this is achieved through checking whether an item is **Consumable** first before checking whether its **Useable.**  

At this stage, my main use case for a `Useable` item is some sort of equipment (which you can see in the code), however this doesn't express the general usage of a Useable type. 

I forsee the need to create items that are  `useable`, but **not equipable**.  (some quest items, reuseable items). 

Valid point though, the logic expressed here can be missleading. I will consider  introducing a mechanism to give more distiction beween  Consumable, Equipable and Useable items, and improve this statement in order to provide clarity in the returnd type from this factory mehod.